packages/os/os.spec

```
@@ -423,6 +425,13 @@ Conflicts: %{_cross_os}bash
 %description -n %{_cross_os}brush
 %{summary}.
 
+%package -n %{_cross_os}whippet
+Summary: A simple dbus-broker launcher
+Provides: %{_cross_os}dbus-broker(launcher) = 0
```

The `:` denotes epoch, otherwise this is providing `Version: 0` for this capability.
```suggestion
Provides: %{_cross_os}dbus-broker(launcher) = 0:
```